server/src/main/java/io/spine/server/commandbus/CommandBus.java

```diff
@@ -206,6 +207,9 @@ protected Ack doPost(CommandEnvelope envelope) {
             if (causedByRejection(e)) {
                 final ThrowableMessage throwableMessage = (ThrowableMessage) cause;
                 final Rejection rejection = toRejection(throwableMessage, envelope.getCommand());
+                final Class<?> rejectionClass = AnyPacker.unpack(rejection.getMessage())
+                                                         .getClass();
+                Log.log().info("Posting rejection {} to RejectionBus.", rejectionClass.getName());
```

Comment:
`Log.log()` is weird.

Also, that's definitely not `info`. It's either "trace" or "debug". I'd think the former.